app/scripts/controllers/threebox.js

```diff
@@ -1,4 +1,9 @@
 import ObservableStore from 'obs-store'
+
+/* eslint-disable import/first,import/order */
```

Comment:
Are these rules disabled here because they conflicted with each other?

Reply:
They're disabled here because they both apply to the following code

Reply:
- `import/first` wants the `require` and `import`s to move
- `import/order` wants the `require` call alone to move

They're both disabled here and re-enabled as soon as ESLint will allow.

Reply:
The e2e tests break if I move the require call so I don't want to move it